crates/oxc_transformer/src/typescript/options.rs

```diff
@@ -44,6 +44,42 @@ pub struct TypeScriptOptions {
     #[serde(default = "default_as_true")]
     pub allow_declare_fields: bool,
 
+    /// When enabled, class fields without initializers are removed.
+    ///
+    /// For example:
+    /// ```ts
+    /// class Foo {
+    ///    x: number;
+    ///    y: number = 0;
+    /// }
+    /// ```
+    /// // transform into
+    /// ```js
+    /// class Foo {
+    ///    x: number;
+    ///    y: number = 0;
+    /// }
+    /// ```
```

Comment:
The output of this example seems incorrect.

Reply:
Oh, sorry, my bad!